README.md

```diff
@@ -121,7 +121,7 @@ Older yarn versions might show an error / hang with the message 'Waiting for the
 
 # Documentation
 
-Find the full documentation for this project at [fbflipper.com](https://fbflipper.com/docs).
+Find the full documentation for this project at [fbflipper.com](https://fbflipper.com/docs/features/index/).
```

Comment:
Hm, I think we would want this to point just to `https://fbflipper.com/` in this case.

Reply:
I have updated the link